daemon/main.c

```diff
@@ -263,20 +263,23 @@ void cancel_main_threads() {
 }
 
 struct option_def option_definitions[] = {
-    // opt description                                    arg name       default value
-    { 'c', "Configuration file to load.",                 "filename",    CONFIG_DIR "/" CONFIG_FILENAME},
-    { 'D', "Do not fork. Run in the foreground.",         NULL,          "run in the background"},
-    { 'd', "Fork. Run in the background.",                NULL,          "run in the background"},
-    { 'h', "Display this help message.",                  NULL,          NULL},
-    { 'P', "File to save a pid while running.",           "filename",    "do not save pid to a file"},
-    { 'i', "The IP address to listen to.",                "IP",          "all IP addresses IPv4 and IPv6"},
-    { 'p', "API/Web port to use.",                        "port",        "19999"},
-    { 's', "Prefix for /proc and /sys (for containers).", "path",        "no prefix"},
-    { 't', "The internal clock of netdata.",              "seconds",     "1"},
-    { 'u', "Run as user.",                                "username",    "netdata"},
-    { 'v', "Print netdata version and exit.",             NULL,          NULL},
-    { 'V', "Print netdata version and exit.",             NULL,          NULL},
-    { 'W', "See Advanced options below.",                 "options",     NULL},
+    // opt description                                        arg name       default value
+    { 'c', "Configuration file to load.",                     "filename",    CONFIG_DIR "/" CONFIG_FILENAME},
+    { 'D', "Do not fork. Run in the foreground.",             NULL,          "run in the background"},
+    { 'd', "Fork. Run in the background.",                    NULL,          "run in the background"},
+    { 'h', "Display this help message.",                      NULL,          NULL},
+    { 'P', "File to save a pid while running.",               "filename",    "do not save pid to a file"},
+    { 'i', "The IP address to listen to.",                    "IP",          "all IP addresses IPv4 and IPv6"},
+    { 'p', "API/Web port to use.",                            "port",        "19999"},
+    { 's', "Prefix for /proc and /sys (for containers).",     "path",        "no prefix"},
+    { 't', "The internal clock of netdata.",                  "seconds",     "1"},
+    { 'u', "Run as user.",                                    "username",    "netdata"},
+    { 'v', "Print netdata version and exit.",                 NULL,          NULL},
+    { 'V', "Print netdata version and exit.",                 NULL,          NULL},
+#ifdef ENABLE_ACLK
+    { 'x', "Socks5 or http proxy for outbound connections",   "proxy",       NULL},
```

Comment:
I don't recall any discussion deciding that we should add this flag. There was definitely some discussion in the estimation meeting but I don't think that we reached a clear decision. What is the use-case for this flag? Why can those use-cases not be achieved using the configuration file? Why are we adding a top-level flag instead of something under `-W`?

Reply:
https://github.com/netdata/netdata/issues/7840

Reply:
It’s not in the definition of the issue. You’ve mentioned in a comment that you want to add it, but where has it been agreed?

Reply:
removed the flag as per slack discussion